admin/bin/qserv-data-loader.py

```diff
@@ -146,8 +144,10 @@ def __init__(self):
                            help='Disable CSS updates.')
 
         group = parser.add_argument_group('Database options', 'Options for database connection')
-        group.add_argument('-H', '--host', dest='mysqlHost', default='localhost', metavar='HOST',
-                           help='Host name for czar mysql server, def: %(default)s.')
+        group.add_argument('-H', '--host', dest='czarHost', default='localhost', metavar='HOST',
+                           help='Host name for czar wmgr service, def: %(default)s.')
+        group.add_argument('-P', '--port', dest='czarPort', default=5012, metavar='PORT_NUMBER', type=int,
+                           help='Port number to use for czar wmgr connection, def: %(default)s.')
```

Comment:
Data loader can run without CSS, I do not want CSS to be prerequisite for data loader (yet). For multi-node data loader we actually can use worker definition from CSS, but this is also not a strict requirement.